Saule/Http/AllowsIncludeAttribute.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System;
+using System.Linq;
+using System.Net.Http;
+using System.Web.Http.Controllers;
+using System.Web.Http.Filters;
+using Saule.Queries.Filtering;
+using Saule.Queries.Including;
+using Saule.Queries.Sorting;
+
+namespace Saule.Http
+{
+    /// <summary>
+    /// Describes how the serializer should treat included resources when no included query is sent.
+    /// </summary>
+    public enum IncludeByDefault
+    {
+        /// <summary>
+        /// All related resources that are present should be included.
+        /// </summary>
+        All,
+
+        /// <summary>
+        /// No related resources should be included.
+        /// </summary>
+        None
+    }
+
+    /// <summary>
+    /// Indicates that the target action supports the use of the include query param.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method)]
+    public sealed class AllowsIncludeAttribute : ActionFilterAttribute
```

Comment:
Moving this out of `AllowsQueryAttribute` seems unrelated to serialization. What was the reason for this change?

Since I already released 1.6, this would be a breaking change. I don't really mind (we can figure out a migration scenario or just release 2.0), but I'm not sure I understand the purpose of this change from the diff.